src/Storage/Synapse/SelectSource.php

```diff
@@ -26,7 +27,7 @@ class SelectSource implements SourceInterface, SqlSourceInterface
     /**
      * @param string[] $columnsNames
      * @param string[]|null $primaryKeysNames
-     * @param array<mixed> $dataTypes
+     * @param array<int|string, Type|int|string|null> $dataTypes
```

Comment:
:D :D hey phpstan, are you happy now ? :D to mi ten mixed dava vetsi smysl... just joke